tests/integration/targets/version_update/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
We need to by default exclude this test from CI jobs - .github/workflows/integ-test.yml.

Reply:
Yes you are right, my bad, I will exclude it from the tests